modules/core/exported/channel.go

```diff
@@ -1,15 +1,5 @@
 package exported
 
-// ChannelI defines the standard interface for a channel end.
-type ChannelI interface {
-	GetState() int32
-	GetOrdering() int32
-	GetCounterparty() CounterpartyChannelI
-	GetConnectionHops() []string
-	GetVersion() string
-	ValidateBasic() error
-}
-
 // CounterpartyChannelI defines the standard interface for a channel end's
 // counterparty.
 type CounterpartyChannelI interface {
```

Comment:
wonder if its possible to remove this also?

Reply:
I think so, decided to tackle one interface at a time. I suspect some other interfaces can be removed as well